src/test/java/org/javarosa/core/model/TriggerableDagTest.java

```diff
@@ -928,6 +928,84 @@ public void relative_paths_in_nested_repeats() throws IOException {
         assertThat(scenario.answerOf("/data/outer[1]/inner[1]/count"), is(intAnswer(2)));
     }
 
+    @Test
+    public void adding_repeat_instance_triggers_triggerables_outside_repeat_that_reference_repeat_nodeset() throws IOException {
```

Comment:
Maybe add a comment with the justification for this test? Something that points out that we didn't know whether the phase 1 of the DAG's algorithm for creating new repeats was justified?

Reply:
Sounds good!